daemon/src/connection.rs

```diff
@@ -483,10 +491,17 @@ impl Actor {
     }
 
     fn handle_measure_pulse(&mut self, _: MeasurePulse) {
+        tracing::trace!("measuring pulse");
+
         if self
             .state
             .disconnect_if_last_heartbeat_older_than(self.heartbeat_timeout)
         {
+            tracing::debug!(
```

Comment:
❓ Should we promote this to `warn` or `info`? It seems like quite a relevant an hopefully not too common event to disconnect from a peer. Or would it be too noisy?

Reply:
Let's make it a `warn` for now - if it spams we can remove / downgrade it again. Maybe this one should stay without the need for debugging.

Reply:
changed to `warn`